src/odemis/util/__init__.py

```diff
@@ -721,3 +721,39 @@ def bindFuture(future, fn, args=(), kwargs=None):
             future.set_exception(e)
     else:
         future.set_result(result)
+
+
+def sum_within_circle(data, i, j, radius):
+    """
+    Assumption: The user selects a "big pixel" of the spectrum data. We treat the width of this "big pixel"
+    as the diameter of a circle.
+    Given that the circle consists of pixels of width 1, we scan the square around the circle and pick
+    only the points in the circle. The mean is calculated by getting the sum of spectrum data corresponding to
+    these points divided by the total number of them.
+
+    param:
+        data (model.DataArray): the data belonging to the selected "big pixel".
+        i: the x coordinate of the selected "big pixel", which consists of pixels.
+        j: the y coordinate of the selected "big pixel", which consists of pixels.
+        radius: the radius of the circle which contains the center of the pixels to be
+        taken into account.
+
+    returns:
+        the mean of the sum of spectrum data that corresponds to points in the circle.
```

Comment:
I'm not completely sure what this function does, but I think the docstring can be made a lot more generic. I don't know what the "big pixel" is, but I also think this function can be explained without using the words "big pixel".